File: jasmine/src/airport.js
function Airport() {
  this.planes = [];
};

Airport.prototype.landing = function(plane) {
  this.planes.push(plane);
};

Airport.prototype.takeOff = function() {
  this.planes.pop();
};

Airport.prototype.weather = function() {
  if(Math.floor(Math.random() * 101) > 85) {
    return "stormy"
  }else{
    return "sunny"
  };
};

File: pre-exercises/python/airport.py
from random import *

planes = []

class Airport():
    def landing(self, plane):
        planes.append(plane)

    def take_off(self):
        planes.pop()

    def weather(self):
        if(randint(1, 100) > 85):
            print("stormy")
        else:
            print("sunny")


# port = Airport()
# port.landing("plane")
# print(planes)
# port.take_off()
# print(planes)
# port.weather()
